### DISCUSSION TITLE
Consulta relacionada al trabajo integrador

### DISCUSSION THREAD

**Discussion by juanmasemper** [frlp-utn-ingsoft/catedra#17] [Q&A] [answered]
Buenas profes! Quería hacer una consulta relacionada al trabajo integrador que a más de uno nos está complicando entender.
En la 2da parte del tutorial de Django en la parte "Para bases de datos distintas a SQLite" dice lo siguiente:
![image](https://github.com/frlp-utn-ingsoft/catedra/assets/136634900/844b6c84-cda0-4534-9c07-ea7a5031c84f)
El problema que nos da es que el mensaje es contradictorio y no entendemos cómo continuar. Nos habla de bd distintas a SQLite y después arranca diciendo "Si está utilizando una base de datos SQLite, asegúrese de que ha creado ..." y al final después pone "Si estás usando SQLite, no es necesario crear nada de antemano...".
Espero su respuesta y estamos en contacto!

Saludos.

**Comment by RodrigoJacznik** [comment, marked as answer]
Buenas, creo que es un error de traducción. En la version en ingles dice "If you’re using a database besides SQLite".
Pueden ignorar esa parte y correr la migración directamente.
